sbt_common/display/src/main/scala/uk/ac/wellcome/display/models/v2/DisplayWorkV2.scala

```diff
@@ -56,10 +56,10 @@ case class DisplayWorkV2(
   @ApiModelProperty(
     value =
       "Relates a work to the general thesaurus-based concept that describes the work's content."
-  ) subjects: List[DisplaySubject] = List(),
+  ) subjects: Option[List[DisplaySubject]] = None,
```

Comment:
Elsewhere I think we’ve had to set the dataType explicitly when we have optional lists, for example:

```scala
dataType = List[uk.ac.wellcome.display.models.v2.DisplaySubject]
```

Have you checked these show up correctly in the Swagger? (Or better, written a test for it!)

Reply:
I don't think they are needed